wgsl/index.bs

```diff
@@ -717,7 +717,7 @@ image_storage_type
      Rg16f  -- Capability: StorageImageExtendedFormats
   | RGBA8UNORM
      Rgba8
-  | RGBA8UNORM-SRGB
+  | RGBA8UNORM_SRGB
```

Comment:
This seems like a good change regardless of this CL, can we pull this out to its own change?